ci/run_tests/run_tests_windows.py

```diff
@@ -1,14 +1,38 @@
 import os
 import argparse
 import subprocess
-import netifaces as ni
+import scapy.arch.windows
+from ipaddress import IPv4Address
 
 TCPREPLAY_PATH = "tcpreplay-4.4.1-win"
 PCAP_FILE_PATH = os.path.abspath(
     os.path.join("Tests", "Pcap++Test", "PcapExamples", "example.pcap")
 )
 
 
+def validate_ipv4_address(address):
+    try:
+        IPv4Address(address)
+        return True
+    except ValueError:
+        return False
+
+
+def get_ip_by_guid(guid):
+    interfaces = scapy.arch.windows.get_windows_if_list()
+    for iface in interfaces:
+        ips = iface.get("ips", [])
+        if not isinstance(ips, list):
```

Comment:
I think this check is redundant, as you use `iface.get("ips", [])` already?

Reply:
removed